src/arch/e2k/entry.c

```diff
@@ -17,6 +17,7 @@ do { \
 
 static int volatile entries_count = 0;
 static int volatile idled_cpus_count = 0;
+static int volatile sync_count = 0;
```

Comment:
it's just needed if log_debug enabled in e2k_context module else output will be wrong because 2 CPUs printing at the same time.
I wrote the comment before the waiting loop.